eZ/Publish/API/Repository/Tests/SearchServiceFieldFiltersSolrTest.php

```diff
@@ -392,7 +387,7 @@ public function testFullTextQueryLanguageAlwaysAvailable( $type = null )
             array(
                 $type => new Criterion\FullText( "one" ),
                 'sortClauses' => array(
-                    new SortClause\Field( "folder", "name", Query::SORT_ASC, "eng-GB" ),
+                    new SortClause\ContentId( Query::SORT_ASC ),
```

Comment:
Unreliable order, because of the bug where prioritized languages are not excluded with always available fallback. Since both eng-GB and eng-US are returned and sort clause does not use language any more, sort value was the same, and test failed in multicore setup.

Note that the bug is fixed in https://github.com/ezsystems/ezpublish-kernel/issues/1298 and tests are again updated. There is a separate issue to write a new set of tests, ATM these are only adapted from the tests for now deprecated behavior, and I don't really care about them much.